examples/animation/animated_fox.rs

```diff
@@ -21,20 +21,43 @@ fn main() {
 }
 
 #[derive(Resource)]
-struct Animations(Vec<Handle<AnimationClip>>);
+struct Animations {
+    animations: Vec<AnimationNodeIndex>,
+    #[allow(dead_code)]
+    graph: Handle<AnimationGraph>,
+}
 
 fn setup(
     mut commands: Commands,
     asset_server: Res<AssetServer>,
     mut meshes: ResMut<Assets<Mesh>>,
     mut materials: ResMut<Assets<StandardMaterial>>,
+    mut graphs: ResMut<Assets<AnimationGraph>>,
 ) {
+    // Build the animation graph
+    let mut graph = AnimationGraph::new();
+    let animations = graph
+        .add_clips(
+            [
+                "models/animated/Fox.glb#Animation2",
+                "models/animated/Fox.glb#Animation1",
+                "models/animated/Fox.glb#Animation0",
+            ]
+            .into_iter()
+            .map(|path| asset_server.load(path)),
+            1.0,
+            graph.root,
+        )
+        .collect();
+
+    graph.save(&mut io::stdout()).unwrap();
```

Comment:
```suggestion
```
this prints the graph to stdout without an explanation, is it still needed here? and it collides with the examples instructions that are printed to stdout